lib/rules/no-extend-native.js

```diff
@@ -1,6 +1,7 @@
 /**
  * @fileoverview Rule to flag adding properties to native object's prototypes.
  * @author David Nelson
+ * @deprecated in ESLint v3.2.0
```

Comment:
Yes, that's what we decided in https://github.com/eslint/eslint/issues/6586#issuecomment-232503727